js/modules/Search/Table.js

```diff
@@ -44,6 +44,19 @@ window.GLPI.Search.Table = class Table extends GenericView {
         super(element_id);
 
         this.shiftSelectAllCheckbox();
+        this.toggleSavedSearch(true);
```

Comment:
I know this is an edge case, but would this force the user to use the "search" button to be able to tranform into a saved search a search operation that was trigerred by accessing the page with an URL containing specific criteria ?

For instance, a user may access a search operation from an intranet link, a dashboard card, a FAQ article, ... and transform it into a saved search.

Reply:
yes, and he may just have to submit once the search, seems ok for me.